src/Middleware/ResponseCaching/ref/Microsoft.AspNetCore.ResponseCaching.netcoreapp3.0.cs

```diff
@@ -31,77 +31,21 @@ public partial class ResponseCachingOptions
 }
 namespace Microsoft.AspNetCore.ResponseCaching.Internal
```

Comment:
Still pubternal? I was expecting all of these to go internal.

Reply:
Ah I missed a few that were under a different directory.

Reply:
Still here?